web/src/main/java/com/decibeltx/studytracker/web/controller/api/StudyAssayController.java

```diff
@@ -83,7 +83,8 @@ public Assay findById(@PathVariable("assayId") String assayId) throws RecordNotF
       if (studyNotebookService == null) {
         throw new RecordNotFoundException("Could not create new entry");
       }
-      studyNotebookService.createAssayNotebookEntry(created, assay.getEntryTemplateId());
+      studyNotebookService.createAssayNotebookEntry(created, assay.getEntryTemplateId(),
+              user.getAttributes().get("benchlingUserId"));
```

Comment:
Should probably add a check that the attribute exists for the user, so you don't get a NullPointerException. This value won't always be set, so it should be okay of the `benchlingUserId` is null.